docker/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+
+services:
+  slurmdbd:
```

Comment:
Can you please update the service name ?

Reply:
OMG - thanks for catching that - too much copy&paste. My bad - fixed now

Reply:
No worries 👍  Thanks